profiler/src/ProfilerEngine/Datadog.Profiler.Native/ClrEventsParser.cpp

```diff
@@ -13,14 +13,14 @@
 #include "OpSysTools.h"
 
 
-const bool LogGcEvents = false;
+const bool LogGcEvents = true;
```

Comment:
will it be always `true` ? if yes, that could be a perf overhead.

Reply:
yes : I need to change it to false after everything is implemented